package.json

```diff
@@ -53,6 +53,7 @@
     "webpack": "1.12.9"
   },
   "dependencies": {
-    "delegate": "git://github.com/vigetlabs/delegate#3.0.1"
+    "delegate": "git://github.com/vigetlabs/delegate#3.0.1",
```

Comment:
Looks like this was just merged upstream and published. Kudos to @greypants for PR'ing that:

https://github.com/zenorocha/delegate/commit/03474cc8f3b8650771f163a514bb6ee861080e19